src/Congo.WebApi/Data/Migrations/20211102090753_AddCartAndCartItem.cs

```diff
@@ -0,0 +1,671 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Congo.WebApi.Data.Migrations
+{
+    public partial class AddCartAndCartItem : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DeleteData(
```

Comment:
Is it intentional that this PR is deleting all the test data?

Reply:
nope i just ran a normal migration, no idea why the test data is there

Reply:
Wish we caught this before. I've figured it out. Since we are using `Guid.NewGuid()` when we generate the products, every time we add a new migration each product is given a new guid.

Seeding the guid for the test data is an option, can do the following, not tested but should take care of it:
in `Congo.WebApi.Extension.ModelBuilderExtensions` add this function:
```c#
// might be other ways to do this but this should be just fine for what we're trying to do
public static Guid StringToGuid(string input)
{
      using MD5 md5 = MD5.Create();
      var hash = md5.ComputeHash(Encoding.Default.GetBytes(input));
      return new Guid(hash);
}
```
Then in `SeedProducts` in our `yield return` change..
```c#
Id = Guid.NewGuid()
```
to
```c#
Id = StringToGuid(name) // name = $"{product} - {i + 1}"
```

Reply:
I tested adding a new migration and then it calls the Update method on each item in the migration instead of adding, is there a way to not have all the seed data pop up in every migration we create?

Reply:
Did my suggestion not work? I think you'd just need to add the one migration the seeded data then it won't delete and re-add the data, with my solution.

An easier route, I think, is doing it like so I mentioned in this comment https://github.com/Code2Gether-Discord/Congo/pull/64#issuecomment-944909688

Reply:
Didn't work, i added my migration then i added another to test it out. Yea probably want to do the seeding that way instead.